meta-mender-clearfog/templates/local.conf.append

```diff
@@ -1,12 +1,32 @@
 # Appended fragment from meta-mender-community/meta-mender-clearfog/templates
 
+# Uncomment the lines below depending on the model
+
+# Clearfog - Base
 MACHINE ?= "clearfog-base"
+MENDER_IMAGE_BOOTLOADER_FILE = "u-boot-clearfog-base.bin"
+
+# Clearfog - Pro
+#MACHINE ?= "clearfog-pro"
+#MENDER_IMAGE_BOOTLOADER_FILE = "u-boot-clearfog-pro.bin"
+
+# Uncomment the line below which matches the storage medium you intend to use
+
+# SD Card
+UBOOT_BINARY ?= "u-boot-spl-sdhc.kwb"
+# SPI Interface
+#UBOOT_BINARY ?= "u-boot-spl-spi.kwb"
+# NAND
+#UBOOT_BINARY ?= "u-boot-spl-nand.kwb"
+# SATA
+#UBOOT_BINARY ?= "u-boot-spl-sata.kwb"
+# eMMC
+#UBOOT_BINARY ?= "u-boot-spl-mmc.kwb"
+# UART (Serial)
+#UBOOT_BINARY ?= "u-boot-spl-uart.kwb"
```

Comment:
Replace this with a link to,

https://github.com/mirzak/meta-clearfog/blob/master/conf/machine/include/clearfog-common.inc#L8-L18

To avoid duplication of the same information.

Reply:
Done